sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java

```diff
@@ -240,64 +240,62 @@ private Response modifyRequest(ObjectNode body) {
     private Optional<String> inject(ObjectNode body) {
         JsonNode specNode = body.path(REQUEST_PATH).path(OBJECT_PATH).path(SPEC_PATH);
 
-        // 获取容器的节点，便于遍历取值
+        // Get the node of the container for traversing the value
         JsonNode containersNode = specNode.path(CONTAINERS_PATH);
 
-        // 缓存每个容器的环境变量
+        // Cache environment variables for each container
         Map<Integer, Map<String, String>> containerEnv = new HashMap<>();
 
-        // 缓存容器的env
+        // Cache env of container
         setEnv(containersNode, containerEnv);
 
-        // 根据labels计算需要新增env
+        // Add env based on labels
```

Comment:
Calculate the env to be added based on the labels.

Reply:
done